2.3.4/bullseye/Dockerfile

```diff
@@ -0,0 +1,217 @@
+#
+# Copyright 2022 Astronomer Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG APT_DEPS_IMAGE="airflow-apt-deps"
+ARG PYTHON_MAJOR_MINOR_VERSION="3.9"
+ARG PYTHON_BASE_IMAGE="python:${PYTHON_MAJOR_MINOR_VERSION}-slim-bullseye"
+
+FROM ${PYTHON_BASE_IMAGE} as airflow-apt-deps
+
+LABEL maintainer="Astronomer <humans@astronomer.io>"
+
+ARG ASTRONOMER_USER="astro"
+ARG ASTRONOMER_UID="50000"
+
+LABEL io.astronomer.docker=true
+LABEL io.astronomer.docker.distro="debian"
+LABEL io.astronomer.docker.module="airflow"
+LABEL io.astronomer.docker.component="airflow"
+LABEL io.astronomer.docker.uid="${ASTRONOMER_UID}"
+
+
+ARG ORG="astronomer"
+
+
+ENV AIRFLOW_HOME="/usr/local/airflow"
+ENV PYTHONPATH=${PYTHONPATH:+${PYTHONPATH}:}${AIRFLOW_HOME}
+
+ENV ASTRONOMER_USER=${ASTRONOMER_USER}
+ENV ASTRONOMER_UID=${ASTRONOMER_UID}
+
+# Need to repeat the empty argument here otherwise it will not be set for this stage
+# But the default value carries from the one set before FROM
+ARG PYTHON_BASE_IMAGE
+ENV PYTHON_BASE_IMAGE=${PYTHON_BASE_IMAGE}
+ARG PYTHON_MAJOR_MINOR_VERSION
+ENV PYTHON_MAJOR_MINOR_VERSION=${PYTHON_MAJOR_MINOR_VERSION}
+ARG PIP_VERSION="21.2.4"
+ENV PYTHON_PIP_VERSION=${PIP_VERSION}
+
+# Make sure noninteractie debian install is used and language variables set
+ENV DEBIAN_FRONTEND=noninteractive LANGUAGE=C.UTF-8 LANG=C.UTF-8 LC_ALL=C.UTF-8 \
+    LC_CTYPE=C.UTF-8 LC_MESSAGES=C.UTF-8
+
+# By increasing this number we can do force build of all dependencies
+ARG DEPENDENCIES_EPOCH_NUMBER="3"
```

Comment:
Does this need to start at 3, or can it start at 1?

Reply:
Yeah. I think it can be 1. Have moved it to 1. copy-paste error :)